frontend/src/lib/components/SelectBox.tsx

```diff
@@ -5,7 +5,7 @@ import { List } from 'antd'
 import { DownOutlined, RightOutlined } from '@ant-design/icons'
 import { ActionType, CohortType } from '~/types'
 import { selectBoxLogic } from 'lib/logic/selectBoxLogic'
-import './SelectBox.scss'
```

Comment:
does this mean we're double importing stuff?

Reply:
Yes, potentially — I can comment this out.